controllers/reconciler.go

```diff
@@ -405,6 +409,9 @@ func secretForTunnel(r GenericTunnelReconciler) *corev1.Secret {
 func deploymentForTunnel(r GenericTunnelReconciler) *appsv1.Deployment {
 	ls := labelsForTunnel(r.GetTunnel())
 	replicas := r.GetTunnel().GetSpec().Size
+	nodeSelector := nodeSelectorsForTunnel(r.GetTunnel())
+	tolerations := r.GetTunnel().GetSpec().Tolerations
+	r.GetLog().Info("NodeSelectors", "NodeSelectors", nodeSelector)
```

Comment:
The config need not be logged back

Reply:
Indeed, change snuck in from testing. Addressed in [2cfbf67](https://github.com/adyanth/cloudflare-operator/pull/98/commits/2cfbf6712d8770deec8ea079b83b3597a19d1f99)